packages/nc-gui-v2/pages/[projectType]/[projectId]/index.vue

```diff
@@ -326,12 +326,11 @@ definePageMeta({
         <DashboardTreeView v-show="isOpen" />
       </a-layout-sider>
     </template>
-
-    <dashboard-settings-modal v-model="dialogOpen" :open-key="openDialogKey" />
-
-    <NuxtPage />
-
-    <GeneralPreviewAs float />
+    <div :key="$route.fullPath">
```

Comment:
A new div could break the structure of the view - a `<template>` tag should work as well. It allows for the key prop also :)

Reply:
Seems `< template>` would break the view

![image](https://user-images.githubusercontent.com/35857179/185651825-04764ceb-ae13-41d9-beae-cb8cd40a8495.png)

Reply:
If it works with the div, go for that then 👍 
Might be that the new div won't be stretching into full container size but I didn't check yet, was just a hunch.